node/clients/search-graphql.ts

```diff
@@ -1,18 +1,22 @@
-import { AppGraphQLClient, IOContext, InstanceOptions } from "@vtex/api";
+import { IOContext, InstanceOptions } from "@vtex/api";
+import { GraphQLServer } from './graphql-server';
+
 import { pathOr } from "ramda";
 
-export class SearchResolver extends AppGraphQLClient {
+const extensions = {
+  persistedQuery: {
+    provider: 'vtex.search-graphql@0.x',
+    sender: 'vtex.search@0.x',
+  },
+}
```

Comment:
@hiagolcm @viniagostini should this PR also follow the new changes made here: https://github.com/vtex-apps/search/pull/56/commits/3b7d604f9fe694e0d6e982b2eb945b141f1e9577?

Reply:
the mistake is in my PR 😅